src/pybind/mgr/mgr_module.py

```diff
@@ -1386,6 +1386,14 @@ def get_perf_schema(self,
         """
         return self._ceph_get_perf_schema(svc_type, svc_name)
 
+    def get_rocksdb_version(self) -> Dict[str, int]:
```

Comment:
Force-pushed a second time because I noticed an extra comma here when there shouldn't have been.

Before: `def get_rocksdb_version(self,) -> Dict[str, int]:`
After: `def get_rocksdb_version(self) -> Dict[str, int]:`

Reply:
def get_rocksdb_version(self) -> int: ...

Reply:
Ack! I thought I had fixed this, but I guess I reverted the changes somewhere. Thanks.

Before: `def get_rocksdb_version(self) -> Dict[str, int]:`
After: `def get_rocksdb_version(self) -> int:`

Fixed it in the other area too:
Before: `def _ceph_get_rocksdb_version(self) -> Dict[str, int]: ...`
After: `def _ceph_get_rocksdb_version(self) -> int: ...`